src/Actions/EnqueueAndInject.php

```diff
@@ -161,4 +161,36 @@ public function bcgov_block_theme_generate_google_ld_json() {
 		// Output the inline script.
 		echo '<script type="application/ld+json">' . wp_json_encode( $json_data ) . '</script>';
 	}
+
+	/**
+     * Generates and outputs the CSS defined by the options.
+     *
+     * Retrieves the mobile breakpoint setting and generates the CSS variable required to enable a custom breakpoint.
+     *
+     * @since 1.3.1
+     * @return void
+     */
+	public function bcgov_block_theme_css_settings() {
+		// Get the current mobile nav breakpoint setting.
+		$mobile_nav_breakpoint = get_option( 'bcgov_mobile_nav_breakpoint_settings', '' );
+
+		// If the breakpoint is not set, use the default value.
+		if ( empty( $mobile_nav_breakpoint ) ) {
+			$mobile_nav_breakpoint = '600';
+		}
+
+		// Output the inline style.
+		echo '<style> /* WordPress hamburger menu override */
```

Comment:
maybe a slightly better way of doing this would have to be to create a css variable for the breakpoint, and including the css within the dist files, one disadvantage of this method, is no caching